Description: Fix definition of BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS for hppa
Author: John David Anglin <dave.anglin@bell.net>
Forwarded: https://github.com/boostorg/math/issues/89
Reviewed-by: Steve Robbins <smr@debian.org>

Note that Boost 1.65.1 appears to have mistakenly commented out the #define.
See github issue 89.  The Debian patch was updated to remove this.  



--- boost1.65.1-1.65.1+dfsg.orig/libs/math/include/boost/math/tools/config.hpp
+++ boost1.65.1-1.65.1+dfsg/libs/math/include/boost/math/tools/config.hpp
@@ -29,9 +29,10 @@
 #include <boost/math/tools/user.hpp>
 
 #if (defined(__CYGWIN__) || defined(__FreeBSD__) || defined(__NetBSD__) \
-   || (defined(__hppa) && !defined(__OpenBSD__)) || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
+   || (defined(__hppa) && !defined(__OpenBSD__) && !defined(__linux__)) \
+   || (defined(__NO_LONG_DOUBLE_MATH) && (DBL_MANT_DIG != LDBL_MANT_DIG))) \
    && !defined(BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS)
-#  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
+#  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
 #endif
 #if BOOST_WORKAROUND(__BORLANDC__, BOOST_TESTED_AT(0x582))
 //
